controllers/feeds/getFeedsV2.js

```diff
@@ -90,12 +91,7 @@ const isValidActivity = async (item, conditions) => {
 
   // filter by threshold
   const threshold = ACTIVITY_THRESHOLD[feed.toUpperCase()];
-  if ((item.final_score || 0) < threshold) {
-    console.log(`final_score under threshold => `, item.final_score);
-    return false;
-  }
-
-  return true;
+  return (item.final_score || 0) >= threshold;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic for checking the threshold has been simplified. This change should not affect the functionality as long as `item.final_score` is always a number or undefined. If there's a chance it could be another falsy value like an empty string, this could introduce bugs.

```diff
- if ((item.final_score || 0) < threshold) {
-   console.log(`final_score under threshold => `, item.final_score);
-   return false;
- }
- return true;
+ return (item.final_score || 0) >= threshold;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->